src/SegmentServiceContract.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace SlashEquip\LaravelSegment;
+
+use SlashEquip\LaravelSegment\Contracts\CanBeIdentifiedForSegment;
+use SlashEquip\LaravelSegment\Contracts\CanBeSentToSegment;
+
+interface SegmentServiceContract
```

Comment:
We'd need to bind this new contract to the original `SegmentService` in the service provider, right?